packages/esm-appointments-app/src/home-appointments/appointments-table.resource.ts

```diff
@@ -0,0 +1,40 @@
+import useSWR from 'swr';
+import { openmrsFetch } from '@openmrs/esm-framework';
+import { AppointmentService, Appointment } from '../types';
+import { useMemo } from 'react';
+import { startDate } from '../helpers';
```

Comment:
Does this work? I don't see startDate being exported from `helpers`

Reply:
There is a helper file on the root directory and that's where we have startDate. And yes it is working, maybe we should look at bringing all helpers in one folder to avoid confusion.